js/build_map.js

```diff
@@ -129,6 +129,11 @@ function fillMap() {
   addCircles(circlesPaths);*/
   updateCircles(activeCategory);
   
+  // manipulate dropdowns
+  // need to figure out how to do this for only mobile views
```

Comment:
I'm working on a switch between buttons on the left and buttons on the bottom, triggered by a resize and (so far, at least) whether the window width is < or > than 425. We could set a global variable for `mobileMode` in there and then tie this line to that variable, once my PR is in.